src/index.html

```diff
@@ -371,7 +371,9 @@ <h2 class="pageItemTitle" style="display: block;">
                         <img width=30px src="" data-bind="attr: { src: Weather.image() }"/>
                     </button>
                     <div id="mapBody" class="card-body p-0 show">
-                        <div id="subregion-travel-buttons" class="btn-group mr-2" role="group" style="width: 100%;" data-bind="visible: SubRegions.getSubRegions(player.region).filter(sr => sr.unlocked()).length > 1">
+                        <div id="subregion-travel-buttons" class="btn-group mr-2" role="group" style="width: 100%;" data-bind="
+                            visible: SubRegions.getSubRegions(player.region).filter(sr => sr.unlocked()).length > 1,
+                            css: { 'slow-flash': player.highestRegion() == player.region && SubRegions.getSubRegions(player.region).filter(sr => sr.unlocked()).length > player.highestSubRegion() + 1 }">
```

Comment:
Would checking against the players current subregion work with this?
```suggestion
                            css: { 'slow-flash': player.highestRegion() == player.region && player.subregion < player.highestSubRegion() }">
```

Reply:
> I think this needs to make use of the following, as we don't have a setter/getter bound for it?

Fixed

> Would checking against the players current subregion work with this?

Then it will only flash, when the player is at an earlier subregion, than they have visited

Reply:
Yeah ignore this sorry, not sure what I was thinking 😅